### PR TITLE
Practical filters added to the UFO filters

### PR DESCRIPTION
**Pull request by emanueleemili**
## Description

(Instructions: this, and all subsequent sections of text should be removed and filled in as appropriate.)
Provide a detailed description of what this PR does.
What bug does it fix, or what feature does it add?
Is a change of answers expected from this PR?

## Definition of Done

What does it mean for this issue to be done?  Is there a specific, measurable, milestone?

### Issue(s) addressed

Link the issues to be closed with this PR
- fixes #<issue_number>

## Dependencies

If there are PRs that need to be merged before or along with this one, please add "waiting for another PR" label and list the dependencies in the other repositories (example below). Note that the branches in the other repositories should have matching names for the automated tests to pass.
Waiting on the following PRs:
- waiting on JCSDA/eckit/pull/<pr_number>
- waiting on JCSDA/atlas/pull/<pr_number>

## Impact

If changes in this PR will affect other repositories, please add a "waiting for other repos" label, and list the repositories that will be affected to the best of your knowledge (example below).
Requires changes in the following repositories:
- [ ] saber
- [ ] ioda
- [ ] ufo
- [ ] ...

If changes in this PR require updating test data on AWS please add an "update test data" label and list the test data that needs to be updated to the best of your knowledge (example below).
Requires updating AWS test data for the following repositories:
- [ ] saber
- [ ] ioda
- [ ] ...

Note: to automatically run saber, ioda and ufo tests with this PR, push a commit containing "trigger pipeline", e.g.:
git commit --allow-empty -m 'trigger pipeline'

### CODE REVIEW

**Review by SamuelTrahanNOAA — COMMENTED** [review]
These changes implement the same feature as PRs #1, #3, #4, #5, #6, and #7. You should discuss with other developers which PR should be merged to the main "develop" branch.

File: test/testinput/qc_practical_boundscheck.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    simulated variables: [variable1, variable2, variable3]
+  obs filters:
+  - filter: Practical Bounds Check        # test min/max value with all variables
+    filter variables:
+    - name: variable1
+    - name: variable2
+    - name: variable3
+    minvalue: 14.0
+    maxvalue: 19.0
+# Compare variables with minvalue/maxvalue
```
Comment (andreapiacentini) [inline test/testinput/qc_practical_boundscheck.yaml:18]: Just an example of a comment. Could be a request of explanations

**Review by andreapiacentini — APPROVED** [review]

**Comment by hga007** [comment]
> These changes implement the same feature as PRs #1, #3, #4, #5, #6, and #7. You should discuss with other developers which PR should be merged to the main "develop" branch.

I don't see the need or room for such a comment!  This was an exercise for all the participants of the 6th JEDI academy as a practice on how to issue a PR for those of us that will like to contribute in the future.  Therefore, many identical PR were expected.  As I understand it, all these will be removed.

**Comment by ezaron** [comment]
test comment

**Pull request closed without merging**